pkg/oc/cli/admin/migrate/migrator.go

```diff
@@ -310,7 +345,7 @@ func (o *ResourceOptions) Complete(f kcmdutil.Factory, c *cobra.Command) error {
 	if o.Unstructured {
 		o.Builder.Unstructured()
 	} else {
-		o.Builder.WithScheme(ocscheme.ReadingInternalScheme)
+		o.Builder.WithScheme(scheme.Scheme, scheme.Scheme.PrioritizedVersionsAllGroups()...)
```

Comment:
I hope this is correct.

Reply:
Yep, this scheme contains our resources + kubectl resources. We have been using it to prioritize getting external, grouped resources across commands here and upstream